regression-tests/test-results/clang-14/mixed-bounds-check.cpp.execution

```diff
@@ -1,2 +1 @@
 Bounds safety violation: out of bounds access attempt detected
-libc++abi: terminating
```

Comment:
This is the platform-dependent message (c.f. the one on Ubuntu below) removed thanks to `std::set_terminate(std::abort);`.